docs/dev/PR.md

```diff
@@ -0,0 +1,66 @@
+# Pull Requests 
+
+Pull Requests in GitHub are a nice means of allowing others to review and test your code. 
+This allows community to know what is coming up in pyDARN and be part of the collaboration.  
+
+
+## Code Check List
+
+Before submitting a PR check the code check list first:
+- [ ] copyright and disclaimer, or modification line is added 
+- [ ] the code installs with python 3.6 and higher 
+- [ ] documentation is updated or in another PR for users 
+- [ ] Follows PEP8 and padas code/doc style
+- [ ] unit tests or various testing has been added/done 
+- [ ] code is on its own *git* branch
+- [ ] all code is pushed to the branch 
+
+## Creating the PR 
+
+Once you have confirmed the above is done you can go to [pyDARN's Pull Requests](https://github.com/SuperDARN/dswg/pulls)
+and click on `New pull request`. If templates are available select which one is most appropriate. Otherwise make your own. 
+
+### Merging Direction 
+
+When in doubt merge to `develop`! Otherwise here are some other options:
+- merge to another branch if you `checkout` from that branch this helps to break things into smaller parts for quicker review and testing
+- merge to `master` **ONLY** if it is a **HOTFIX** or **RELEASE** branch. 
+- In some cases, documentation can be merged to `master` if it is fixing some typos. 
+
+### Writing a PR
+
+Follow the template to fill out the required sections if not here is a check list to ensure you have all the information: 
+- [ ] Informative title 
+- [ ] Description of the changes you made 
+- [ ] Scope on what people should focus on 
+- [ ] How to install the code if there is any changes 
+- [ ] How to test the code and expected output 
```

Comment:
Including code snippet examples?

Reply:
```suggestion
- [ ] Code sippets of how to test the code and expected output/plots
```

Reply:
```suggestion
- [ ] Code fragments showing how to test the code and expected output/plots
```
Sorry, didn't do a suggestion in the first place because a better word than snippet eluded me for a while :P